commcare_connect/reports/views.py

```diff
@@ -419,3 +418,109 @@ def dashboard_stats_api(request):
             "percent_verified": f"{percent_verified:.1f}%",
         }
     )
+
+
+@login_required
+@user_passes_test(lambda u: u.is_superuser)
+def dashboard_charts_api(request):
+    filterset = DashboardFilters(request.GET)
+    queryset = UserVisit.objects.all()
```

Comment:
How confident are you that you want `UserVisit` rather than `CompletedWork` (user visits can be combined to form a single completed work, which is the unit of payment). I think we typically view Completed Works as a unit of delivery instead of the form, since it can account for things a visit that includes 2 forms filled out at the same time that together mark a delivery.

Reply:
It's all using the same base queryset which I got from the superset dashboard's SQL, so I think it's right? Or at least, it's consistent with the superset view and itself.

But can definitely revisit this logic once the dashboard has been vetted by the team more.